src/main/java/seedu/address/logic/Messages.java

```diff
@@ -45,7 +45,11 @@ public static String format(Person person) {
                 .append("\n Diagnosis: ")
                 .append(person.getDiagnosis())
                 .append("\n Medication: ")
-                .append(person.getMedication());
+                .append(person.getMedication())
+                .append("\n\n Notes: ")
+                .append(person.getNotes().toString().isEmpty() ? "-" : person.getNotes())
+                .append("\n\n Appointment: ")
+                .append(person.getAppointment() == null ? "-" : person.getAppointment().toString());
```

Comment:
Implementation looks good, maybe can extract "-" for all empty string in the future

Reply:
That's a great idea, we can look into that when we make diagnosis and medication non compulsory in `add command`, then it shows the "-" 😄